haiku-apps/einsteinium/einsteinium-1.3.0.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="Monitor applications and system services"
+DESCRIPTION="Einsteinium provides smarter monitoring of applications and \
+system services for Haiku. It can restart applications and system services \
+that were quit or crashed, gather statistics on application usage and \
+provides customizable ranked lists of applications."
+HOMEPAGE="https://sourceforge.net/projects/esforhaiku/"
```

Comment:
You may want to point the homepage to https://github.com/Perelandra0x309/einsteinium as well.
You can even create a wiki and simple homepage at github, see e.g. http://humdingerb.github.io/clipdinger/